src/main/kotlin/org/kamiblue/client/module/modules/movement/Strafe.kt

```diff
@@ -57,6 +57,7 @@ internal object Strafe : Module(
     private val strafeTimer = TickTimer(TimeUnit.TICKS)
 
     /* If you skid this you omega gay */
+    /* ^ ez ban for sexual harassment I'm not gay tho*/
```

Comment:
😳 i think this violates the lgpl

Reply:
so nobody can stop sexual harassment :sob: